test/locale.js

```diff
@@ -0,0 +1,154 @@
+/* eslint-env jest */
+import React from 'react'
+import {
+  render,
+  getByText,
+  getByTitle,
+  fireEvent,
+  waitFor
+} from '@testing-library/react'
+import '@testing-library/jest-dom/extend-expect'
+
+import Uploader from '../src/uploader'
+
+describe('Uploader', function () {
+  it('should change locales dynamicaly', async function () {
+    const { container, rerender, unmount } = render(
+      <Uploader publicKey='demopublickey' locale='en' />
+    )
+
+    expect(getByText(container, 'Choose a file')).toBeInTheDocument()
+
+    rerender(<Uploader publicKey='demopublickey' locale='ru' />)
+
+    fireEvent(
+      getByText(container, 'Choose a file'),
+      new window.MouseEvent('click', {
+        bubbles: true,
+        cancelable: true
+      })
+    )
+
+    expect(
+      getByText(container.nextSibling, 'Выберите локальный файл')
```

Comment:
I think a better way, it's export locale from uploader and checks by it. 

`export ruTranslations from '${path}/ru.js'`

And expect after reinit.

window.UPLOADCARE_LOCALE.equal(ruTranslations)

What do you think?

Reply:
I don't like hardcode translations too, I will solve this problem in future